src/api/posts.ts

```diff
@@ -0,0 +1,25 @@
+import { Post } from '../react-app-env';
+
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getPosts(userId: string): Promise<Post[]> {
```

Comment:
Missing requirement: Create an `/src/api/posts.ts` and add a method `getUserPosts(userId)` there.
```suggestion
export async function getUserPosts(userId: string): Promise<Post[]> {
```